README.md

```diff
@@ -183,28 +183,16 @@ Example of how to log:
 Log("This is the message to be logged.", onLevel: .info)
 ```
 
-### Concurrency modes
+### Logging execution
 
-`LoggerManager` supports 3 different concurrency modes at the moment. By using `loggingConcurrencyMode` parameter when initializing `LoggerManager`, a user can set specific concurrency mode. Default value is `asyncSerial`.
-
-#### `syncSerial`
-
-Logging task is dispatched synchronously on a custom serial queue, where all loggers perform their tasks serially one by one.
-
-![syncserial](https://user-images.githubusercontent.com/2511209/33495947-a2f21ca2-d6c8-11e7-9082-f841ef074012.png)
+`LoggerManager` has its own style of executing tasks. It's done in form of `asyncSerial` style, that works on background thread without disrupting the main thread.
```

Comment:
Nemas cleny, `in the form of` a `works on the background thread`. Navic tam mas jen pulku pribehu, a to `async`. Napsal bych tam jeste vetu k ty druhy pulce pribehu, tj. `serial`. Logging into individual loggers is done in serial fashion, thus one by one. Nebo neco takovyho...